sdk/profiles/src/main/java/io/opentelemetry/sdk/profiles/data/ValueTypeData.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.profiles.data;
+
+import javax.annotation.concurrent.Immutable;
+
+/**
+ * ValueType describes the type and units of a value, with an optional aggregation temporality.
+ *
+ * @see "pprofextended.proto::ValueType"
+ */
+@Immutable
+public interface ValueTypeData {
+
+  /** Index into string table. */
+  long type();
+
+  /** Index into string table. */
+  long unit();
```

Comment:
seems like it should be an enum?

Reply:
Again, would need semantic conventions to specify the valid values. Basically we're in the same situation here as for Location.getTypeIndex

Reply:
Can't do enums yet, but work is underway to define common values, which may then become String constants and perhaps eventaully enums. Can revisit the interface when that spec work is further along.
https://github.com/open-telemetry/opentelemetry-proto/issues/555